src/server/game/Movement/MotionMaster.cpp

```diff
@@ -33,9 +32,18 @@
 #include "MoveSpline.h"
 #include "MoveSplineInit.h"
 
-inline bool isStatic(MovementGenerator *mv)
+inline bool IsStatic(MovementGenerator *mv) { return (mv == &si_idleMovement); }
```

Comment:
and cant be delcared in the header, it'd need to add an include or a forward declaration of the movement generator